snaps-openstack/packages/source/inventory/multinode

```diff
@@ -30,7 +30,7 @@
 10.197.101.25 
 10.197.101.26 
 10.197.101.23
-# ansible_connection=ssh ansible_ssh_user=root ansible_ssh_pass=abc123
+# ansible_connection=ssh ansible_ssh_user=root ansible_ssh_pass=abc123 
```

Comment:
This list is commented out.  Is it needed and why wa a space added to the end of the line?

Reply:
Hi Randy,
The list is not required and the line should be deleted.
Ideally, we shouldn't be committing this file at all. AOIP SNAPS-OpenStack generates this file everytime deployment is done, so we should modify our code to delete all dynamically generated file.